Components/Data/Article/Entity/ExportArticle.php

```diff
@@ -34,16 +39,17 @@ public function getId(): int
 
     public function toArray(): array
     {
-        $article = $this->detail->getArticle();
+        $article      = $this->detail->getArticle();
+        $translations = $this->translationService->getArticleTranslation($article->getId());
 
         $data = [
             'ProductNumber' => (string) $this->detail->getNumber(),
             'Master'        => (string) $article->getMainDetail()->getNumber(),
-            'Name'          => (string) $article->getName(),
+            'Name'          => (string) $translations['name'] ?: $article->getName(),
             'EAN'           => (string) $this->detail->getEan(),
             'Weight'        => (float) $this->detail->getWeight(),
-            'Description'   => (string) $article->getDescriptionLong(),
-            'Short'         => (string) $article->getDescription(),
+            'Description'   => (string) $translations['description'] ?: $article->getDescriptionLong(),
```

Comment:
should be `'Description'   => (string) $translations['descriptionLong'] ?: $article->getDescriptionLong(),`

Reply:
fixed